src/main/java/Soot.java

```diff
@@ -0,0 +1,128 @@
+import java.util.Scanner;
+
+public class Soot {
+    public static int listCounter = 0;
+    static final int LIST_SIZE = 100;
+    public static void main(String[] args) {
+        String line;
+        String lowerCase;
+        Scanner in = new Scanner(System.in);
+        Task[] taskList = new Task[LIST_SIZE];
+
+        boolean isBye = false;
+
+        drawLine(); //initial greeting
+        greetUser();
+
+        while (!isBye) {
+            line = in.nextLine(); //user input
+            drawLine();
+            lowerCase = line.toLowerCase();
+            isBye = verifyInput(taskList, line);
+        }
+    }
+
+    public static boolean verifyInput(Task[] taskList, String input) {
+        String inputCommand = input.toLowerCase();
+        String inputTask = input;
+        if (inputCommand.startsWith("done")) {
+            inputCommand = "done";
+            inputTask = input.substring(5);
+        } else if (inputCommand.startsWith("unmark")) {
+            inputCommand = "unmark";
+            inputTask = input.substring(7);
+        } else if (inputCommand.startsWith("todo")) {
+            inputCommand = "todo";
+            inputTask = input.substring(5);
+        } else if (inputCommand.startsWith("deadline")) {
+            inputCommand = "deadline";
+            inputTask = input.substring(9);
+        } else if (inputCommand.startsWith("event")) {
+            inputCommand = "event";
+            inputTask = input.substring(6);
+        }
```

Comment:
This method has more than 30 LoC, it would be better if you break it into two methods, one to parse the input, and one to handle each command.